port/webhook/webhookResourceToPortBody.go

```diff
@@ -0,0 +1,114 @@
+package webhook
+
+import (
+	"context"
+
+	"github.com/port-labs/terraform-provider-port-labs/internal/cli"
+)
+
+func webhookResourceToBody(ctx context.Context, state *WebhookModel) (*cli.Webhook, error) {
```

Comment:
the name doesn't match the file name

either add `Port` to the function or remove `Port` from the filename